processing/data_collection/gazette/spiders/pa_belem.py

```diff
@@ -0,0 +1,64 @@
+import scrapy
+import json
+import dateparser
+from datetime import datetime
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class BelemSpider(BaseGazetteSpider):
```

Comment:
This code is not following the code style defined by Black (https://github.com/okfn-brasil/diario-oficial/blob/master/CONTRIBUTING.md#automated-code-formatting). Running black will fix these problems.